Makefile

```diff
@@ -79,6 +79,10 @@ endif
 	# Diff of uncommitted changes for running locally
 	git diff HEAD -- rasa | poetry run flake8 --select D --diff
 
+lint-security:
+	poetry run pip install bandit\>=1.6.2,\<2.0
```

Comment:
it's quicker to install it this way in CI (you do not have to run `make install`) but it's harder to maintain. Can we specify this requirement inside `pyproject.toml`?

Reply:
hmm could we use something like https://python-poetry.org/docs/pyproject/#extras for this?

Reply:
no not really, because when you run `poetry install --extras stuff` it installs all the dependencies + the extra ones, which in our case doesn't help 😕

Reply:
aha I see, that's annoying

Reply:
why not having it in the `dev-dependencies`?

Reply:
that was also a possibility - do you think it would be useful to run it locally? We'd be sacrificing some performances in the CI but I think we can live with that if it's actually useful to have it locally.

Reply:
Ideally it's restored from cache anyway so I don't see the performance impact on the CI as critical.
We run all other linters locally, so I think this one should be set up similarly.

Reply:
ok gotcha, will go with your reco then. I don't know why exactly I thought I could make an exception of it to the detriment of easy dep management...